lib/utils/sbom-cyclonedx.js

```diff
@@ -31,19 +30,18 @@ const cyclonedxOutput = ({ npm, nodes, packageType, packageLockOnly }) => {
   const childNodes = nodes.filter(node => !node.isRoot && !node.isLink)
   const uuid = crypto.randomUUID()
 
-  const deps = []
-  const seen = new Set()
-  for (let node of nodes) {
-    if (node.isLink) {
-      node = node.target
```

Comment:
What was this doing before that we aren't needing now?  It's unclear to my how this change removes the need to set the target here.

Reply:
Previously, this logic was used to de-dupe link nodes from their corresponding target nodes. When there is a linked dependency you'd end-up with something like the following in the list of nodes:

```json
[
  {
    "name": "chai",
    "version": "1.0.0",
    "location": "node_modules/chai",
    "path": "/private/tmp/sbom/node_modules/chai",
    "realpath": "/private/tmp/chai",
    "resolved": "file:../../chai",
    "edgesIn": {},
    "target": {
      "name": "chai",
      "version": "1.0.0",
      "location": "../chai",
      "path": "/private/tmp/chai",
      "edgesOut": {},
      "children": {}
    },
  },
  {
    "name": "chai",
    "version": "1.0.0",
    "location": "../chai",
    "path": "/private/tmp/chai",
    "edgesOut": {},
    "children": {}
  }
]
```

Now, since we're deduplicating ALL of the nodes based on their name/version this is no longer needed -- the link and target or handled as part of the overall deduplication of the list.